lang/ru/auth/login.php

```diff
@@ -1,8 +1,9 @@
 <?php
 
 return [
+    'title_page' => 'Вход',
     'title' => 'Вход',
-    'email' => 'Электронная почта',
+    'email' => 'Email или телефон',
```

Comment:
We will not do phone authentication
```suggestion
    'email' => 'Электронная почта',
```